### DISCUSSION TITLE
GETTING ERROR WHILE INSTALLING WITH COMPOSER

### DISCUSSION THREAD

**Discussion by SheharyarSMAdil** [kimai/kimai#2333] [Questions & Answers]
![Screenshot (200)](https://user-images.githubusercontent.com/41816068/107367679-9abcb900-6b05-11eb-80d9-4fe53a593efd.png)

Getting this error while I am trying to install the package dependencies on Plesk.
please tell me if I am doing something wrong.. Thanks in advance.

**Comment by kevinpapst** [comment]
Your Capslock seems to be broken... you are using the wrong PHP version, switch to PHP >= 7.2.9
